docs/user-guide/measure.md

```diff
@@ -78,6 +78,9 @@ From this step the creation process is the same described in the [Annotations se
 
 When the [3D Navigation](navigation-toolbar.md#3d-navigation) is enabled, the user has the ability to perform distance, area, point coordinates, height from terrain, angle and slope measurements on the 3D map.
 
+!!! note
```

Comment:
!!! note
    The MapStore 3D works on top of CesiumJS mapping library where the ellipsoidal coordinate system is used to represents mapping features. For this reason all measurements performed with the MapStore measurement tool should be interpreted as having a height calculated above the ellipsoid.